.github/workflows/ci.yml

```diff
@@ -43,23 +48,18 @@ jobs:
         run: "cargo test"
 
   maven:
-    name: "Execute build, run tests (Java ${{ matrix.java-version }})"
+    name: "Unit Tests (Java ${{ matrix.java-version }})"
     runs-on: ubuntu-22.04
     strategy:
       fail-fast: false
       matrix:
         java-version:
           - 8
-          - 11
-    env:
-      HEROKU_API_KEY: ${{ secrets.HEROKU_API_KEY }}
```

Comment:
Is this account now completely unused? If so, just checking that decommissioning it is on your todo? :-)
https://github.com/heroku/languages-team/blob/main/guides/create_test_users_for_ci.md#known-usernames

Reply:
Nope, the new integration tests still use those :) Previously, we had some integration tests that ran with `./mvnw test` which is no longer the case so I removed the env here.